viewer/packages/tools/src/TimeLine/TimeLineKeyframe.ts

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { NodeCollectionBase, NodeAppearance, DefaultNodeAppearance } from '@reveal/core/src';
+import { Cognite3DModel } from '@reveal/core';
+
+/**
+ * Timeline Key Frames contains parameters to access Nodes, Styles for the Timeline
+ */
+export class TimeLineKeyframe {
+  private readonly _date: number;
+  private readonly _model: Cognite3DModel;
+  private readonly _nodeCollection: NodeCollectionBase;
+  private readonly _nodeAppearance: NodeAppearance;
+
+  constructor(
+    model: Cognite3DModel,
+    date: number,
+    nodeCollection: NodeCollectionBase,
+    nodeAppearance?: NodeAppearance
+  ) {
+    this._model = model;
+    this._date = date;
+    this._nodeCollection = nodeCollection;
+    this._nodeAppearance = nodeAppearance!;
+  }
+
+  /**
+   * Get date of the TimeLineFrame
+   * @returns date
+   */
+  public getTimeLineFrameDate() {
```

Comment:
```suggestion
  public getTimeLineFrameDate(): Date {
```

Try to add return types to the functions as it makes life easier when reading the code.